### PR TITLE
 Replace chai.should with chai.expect

### LINKED ISSUE #1687: Replace chai.should with chai.expect.
Throughout the codebase we're using `chai.should`, which patches _all_ objects so that they have the `should` property (this is done [here](https://github.com/OpenZeppelin/openzeppelin-solidity/blob/cf7375d6b873afc9f705e329db39e2ef389af9d2/truffle-config.js#L1)). 

It'd be great to get rid of that, and switch to `expect`, which is not only simpler due to it just requiring to import a function, but also has arguably clearer semantics (and prevents ugly hacks like `(await promise()).should.bla.bla.bla`). This conversion should be rather straightforward, with most `obj.should.equal` being replaced for `expect(obj).to.equal`, though we'd also have to add `const { expect } = require('chai')` at the top of the relevant files.

### PR DESCRIPTION
**Pull request by RCYP**
Fixes #1687 

Converted all cases of .should to expect and added 'const { expect } = require('chai');' to relevant files.
Did not touch any other files except the test files.

### CODE REVIEW

**Comment by nventuro** [comment]
Woot, thank you very much for this @RCYP! Note that there are multiple tests failing due to usage of `exxpect` instead of `expect`.

**Comment by nventuro** [comment]
Note that @alanarvelo is working on a very similar thing at https://github.com/OpenZeppelin/openzeppelin-solidity/pull/1804

**Comment by RCYP** [comment]
idk why the build is failing.. i've checked out from the latest master branch and put the test cases but it still fails.. seems @alanarvelo has got it so you can use his instead.

**Comment by alanarvelo** [comment]
Hi @RCYP sorry to have jumped in, I thought you had paused your effort. 
In the _Details_ section provided by Travis CI it says the unit tests and linting tests are failing. 

You can prevent this from happening by running those in your local environemnt by running `truffle test file/you/changed` and `npm lint` respectively, every time you make changes to a file.

**Comment by RCYP** [comment]
the coverage check is failing at Address.sol for some reason.. i'm using the same file from the master. any idea what the issue might be?

File: test/utils/Address.test.js [hunk 1]
```diff
@@ -1,37 +1,21 @@
-const { constants } = require('openzeppelin-test-helpers');
```
Comment (nventuro) [inline test/utils/Address.test.js]: @RCYP it looks like you've mistakenly removed some of the tests here during the merge, which were added in #1773

Comment (RCYP) [reply test/utils/Address.test.js]: Thanks

**Comment by frangio** [comment]
Lovely! Thank you @RCYP!

**Pull request merged**